### PR TITLE
[ci_updates] pylint + Python3 compatibility

### PR DESCRIPTION
**Pull request by fmessmer**
@LoyVanBeek @benmaidel 
this is needed for https://github.com/mojin-robotics/mojin_flexbe/pull/46 to pass...there, it's already used in the rosinstall: https://github.com/mojin-robotics/mojin_flexbe/pull/46/commits/3e4b1e87708037170c15add71925f87b53099cc4

please review

### CODE REVIEW

File: flexbe_core/src/flexbe_core/behavior.py [hunk 1]
```diff
@@ -262,8 +262,8 @@ def _set_typed_attribute(self, name, value):
         if not isinstance(value, type(attr)):
             if type(attr) is int:
                 value = int(value)
-            elif type(attr) is long:
```
Comment (LoyVanBeek) [inline flexbe_core/src/flexbe_core/behavior.py:265]: <strike>
Not sure this is backwards compatible to the extend we want. 
In python2, like C++, a `long` is different from an `int`. In Py3, they're the same. </strike>

This is fine: https://www.python.org/dev/peps/pep-0237/

Comment (LoyVanBeek) [reply flexbe_core/src/flexbe_core/behavior.py:265]: We might actually remove this elif-blocks though

**Review by LoyVanBeek — CHANGES_REQUESTED** [review]
1 small change needed, LGTM otherwise. 
However, we should make a PR to upstream for @pschillinger to review as well quickly

**Comment by fmessmer** [comment]
there already is a PR to `team-vigir`: https://github.com/team-vigir/flexbe_behavior_engine/pull/102

**Comment by fmessmer** [comment]
got merged to the upstream/vigir repo....so I updated our `develop` branch

**Pull request merged**